.github/actions/Ubuntu-build/debian/README.Debian

```diff
@@ -0,0 +1,6 @@
+imagemagick-7.1.0-16 for Debian
+---------------------------
+
+<possible notes regarding this package - if none, delete this file>
```

Comment:
can we just delete this file?

Reply:
Added a note in this file, hence keeping the file.